data-prepper-plugins/opensearch/src/main/java/org/opensearch/dataprepper/plugins/sink/opensearch/OpenSearchSink.java

```diff
@@ -615,6 +615,7 @@ private void closeFiles() {
   public void shutdown() {
     super.shutdown();
     closeFiles();
+    openSearchClient.shutdown();
```

Comment:
What does this shutdown api do ? Does it close the client or shutdown the node ?

Reply:
It just shuts down the client to the opensearch. We create it again when we do `doInitialize`. So, shutting down is the correct thing to do.